app/helpers/quick_search/search_helper.rb

```diff
@@ -1,2 +1,41 @@
 module QuickSearch::SearchHelper
+    def stripcharacter(string, character)
```

Comment:
Try to keep with the same case on method and variable names. For regular variables and method names, underscore case is best practice.

Reply:
There are more instances of this in the rest of the pull request that should be fixed too.